spec/install_spec.sh

```diff
@@ -158,6 +158,8 @@ Describe "./install.sh"
 
       It 'does not create directory'
         When call fetch "http://repo.test/b3d5591.tar.gz" "$TMPBASE/error"
+        # TODO: not working, and I don't know why:
+        # The directory "$TMPBASE/error" should not exist
         The directory "$TMPBASE/error" should not be exist
```

Comment:
@ko1nksm I went back to using the old matchers for 3 test assertions - I don't know why it did not work with the new one :( 

The dedicated tests in stat_spec.sh show that it is working :confused: